### DISCUSSION TITLE
React Native - Continue button on MFA email input page unresponsive - 400 error "register v2: error while sending recovery email"

### DISCUSSION THREAD

**Discussion by miroosama** [Web3Auth/Web3Auth#1233] [Help]
I enter the email for the account and click Continue but the button is unresponsive. MFA has worked previously with the same setup, however this issue has now persisted for a few days

<img width="381" alt="Screen Shot 2023-01-20 at 11 22 16 AM" src="https://user-images.githubusercontent.com/32473240/213752658-3b246a7d-7b58-44ad-8e95-b205d948ccd6.png">

**Comment by VoR0220** [comment]
Also running into this too on multiple areas. Just starts to load and then basically dies.

**Comment by VoR0220** [comment]
Hey dude. So I have been working on this for a while with my coworker and I think we just cracked the code as to why it's not working (though this should really be fixed on web3auth's end). We got rid of the logo and defaulted on the language field (this is not a good solution) and managed to get it to pass. Seems to fail on the email saying that the name can be too long or that the language isn't supported. This needs to be fixed in a subsequent release me thinks.

**Reply by ratribiana** [reply]
Hey mate, thanks for the work around. I have a question anyway. Have you tried to use the self-host tkey?

**Reply by miroosama** [reply]
Nice! Unfortunately for me I hadnt used whitelabel yet so was default. Also tried with whitelabel and defaulting language but the issue persists

**Reply by pmn4** [reply]
fwiw, @VoR0220 and I were able to work around it using raw requests (fetch/curl), which is obviously not a solution. we were _not_ able to get around the issue using web3auth libraries, as they default some of the request data... `language`, for example.

**Reply by miroosama** [reply]
ahh gotcha! Thank you yea that makes sense considering the error is 400

<img width="999" alt="Screen Shot 2023-01-26 at 11 48 06 AM" src="https://user-images.githubusercontent.com/32473240/214897639-ac83c10b-f42a-45ad-a449-b6da5eb18beb.png">

**Reply by pmn4** [reply]
it's this code:
<img width="816" alt="image" src="https://user-images.githubusercontent.com/1158287/214899646-d9c09b85-32f4-4a4d-9547-411cbdbbd763.png">
here is a response from a request using only the defaults:
```
{
    "statusCode": 400,
    "error": "Bad Request",
    "message": "Validation failed",
    "validation": {
        "body": {
            "source": "body",
            "keys": [
                "language"
            ],
            "message": "\"language\" is not allowed"
        }
    }
}
```

**Reply by pmn4** [reply]
even setting `language` to `''` (which would skip the defaulting to `en`) gives the same error

**Reply by VoR0220** [reply]
> Hey mate, thanks for the work around. I have a question anyway. Have you tried to use the self-host tkey?

Not yet, but it looks like we're about to go down that rabbithole. Have you tried and run into issues?

**Reply by miroosama** [reply]
@pmn4 @VoR0220 Thank you for the input on this! Their fix is working

**Comment by arch1995** [comment]
We have released a fix for this issue. Please try again

**Reply by pmn4** [reply]
fwiw, I just tried to log into the web3auth dashboard, got prompted to set up MFA, and got this same experience. Then tried again and was no longer prompted to MFA.
It is also happening in my local development environment. I filed a different discussion ticket for it: #1362

**Reply by miroosama** [reply]
Same, I had to temporarily disable MFA but I need it asap
